zebrad/tests/acceptance.rs

```diff
@@ -351,3 +354,69 @@ fn valid_generated_config(command: &str, expected_output: &str) -> Result<()> {
 
     Ok(())
 }
+
+#[test]
+fn listener_address_test1() -> Result<()> {
```

Comment:
Please name tests based on what they test, rather than numbers.

It's easier to read the test output if each test has a specific name.

```suggestion
fn listener_address_test_valid_hostname() -> Result<()> {
```